composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "raulfraile/ladybug": "~1.0.8",
-        "phpspec/prophecy": "~1.3.1"
+        "mikey179/vfsStream": "~1"
```

Comment:
Unfortunately this is not possible for ConfigFactory as `tempname()` (used `Filesytem::dumpFile()`) always uses the system temp-path.

Wait :D I can set the TEMP folder to the url of the vfsStream :flushed: :smile: :smiling_imp: